### ISSUE TITLE
Production usage

### ISSUE THREAD

**Issue by ghost** [Spinoco/fs2-kafka#11]
Hi 
I wanted to ask, if it is save to use it in production? 
Are you using in production?
Thanks

**Comment by pchlupacek** [comment]
@kostonstyle yes, the fs2-kafka is used currently in production, currently against kafka 11.x series cluster with about 90 JVMs and 3 node ensemble.

**Comment by ghost** [comment]
Does it support kafka 1.0?

**Comment by pchlupacek** [comment]
Kafka 1.0 will be supported in 0.4.1, I hope that this will see the light in like month or so.
